src/style.css

```diff
@@ -1 +1,46 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src:
+    url(../src/fonts/Avenir-Book.ttf) format('book'),
+    url(../src/fonts/Avenir-Heavy.ttf) format('heavy');
```

Comment:
The `format` descriptors in the `@font-face` rule are incorrect. The `format` should specify the font format like 'truetype', 'opentype', etc. For example, use `format('truetype')` instead of `format('book')` or `format('heavy')`.